src/infix-longscale.ts

```diff
@@ -0,0 +1,52 @@
+
+// Copy-pasted from standard.ts
+const ABBREVIATIONS = [
+  ",", "K", "M", ".", "B", ".", "T", ".", "Qa", ".", "Qt", ".", "Sx", ".",
+  "Sp", ".", "Oc", ".", "No", ".", "Dc", ".", "UDc", ".", "DDc",
+  "TDc", ".", "QaDc", ".", "QtDc", ".", "SxDc", ".", "SpDc", ".", "ODc", ".",
+  "NDc", ".", "Vg", ".", "UVg", ".", "DVg", ".", "TVg",
+  "QaVg", ".", "QtVg", ".", "SxVg", ".", "SpVg", ".", "OVg", ".", "NVg", ".",
+  "Tg", ".", "UTg", ".", "DTg", ".", "TTg", ".", "QaTg",
+  "QtTg", ".", "SxTg", ".", "SpTg", ".", "OTg", ".", "NTg", ".", "Qd", ".",
+  "UQd", ".", "DQd", ".", "TQd", ".", "QaQd", ".", "QtQd",
+  "SxQd", ".", "SpQd", ".", "OQd", ".", "NQd", ".", "Qi", ".", "UQi", ".",
+  "DQi", ".", "TQi", ".", "QaQi", ".", "QtQi", ".", "SxQi",
+  "SpQi", ".", "OQi", ".", "NQi", ".", "Se", ".", "USe", ".", "DSe", ".",
+  "TSe", ".", "QaSe", ".", "QtSe", ".", "SxSe", ".", "SpSe",
+  "OSe", ".", "NSe", ".", "St", ".", "USt", ".", "DSt", ".", "TSt", ".",
+  "QaSt", ".", "QtSt", ".", "SxSt", ".", "SpSt", ".", "OSt",
+  "NSt", ".", "Og", ".", "UOg", ".", "DOg", ".", "TOg", ".", "QaOg", ".",
+  "QtOg", ".", "SxOg", ".", "SpOg", ".", "OOg", ".", "NOg",
+  "Nn", ".", "UNn", ".", "DNn", ".", "TNn", ".", "QaNn", ".", "QtNn", ".",
+  "SxNn", ".", "SpNn", ".", "ONn", ".", "NNn", ".", "Ce"
+];
+
+
+import { AbstractInfixNotation } from "./infix-abstract";
+import Decimal from "break_infinity.js";
+
+// Name comes from https://en.wikipedia.org/wiki/Long_and_short_scales
+export class InfixLongScaleNotation extends AbstractInfixNotation {
+  public get name(): string {
+    return "Infix long scale";
```

Comment:
This should probably be called Infix standard for consistency with the notation called standard.

Reply:
Hm , I see you address that. Right long scale and short scale are things. Long scale should probably be added as a separate notation (probably by me), called long scale, and then this can be called Infix long scale. The other one should be Infix standard though.

Reply:
In retrospect since it's a community notation and there aren't backwards compatibility issues, I think we can call the other one Infix short scale. So basically ignore this set of comments.